README.rst

```diff
@@ -615,6 +615,11 @@ Choose from the list of available fixers:
                         closing semicolon are
                         prohibited.
 
+* **native_function_casing** [contrib]
```

Comment:
But in the other function-replacing fixers the change could change code behavior. Here not.
So strictly speaking it is not risky.